dts/msc-sm2-imx6x.dtsi

```diff
@@ -391,8 +391,7 @@
 			regulator-name = "usb_h1_vbus";
 			regulator-min-microvolt = <5000000>;
 			regulator-max-microvolt = <5000000>;
-			gpio = <&gpio1 0 0>;
-			enable-active-high;
+			regulator-always-on;
```

Comment:
This pin, MX6QDL_PAD_GPIO_0__GPIO1_IO00 (&gpio1 0), is connected to P67 on the SMARC connector. And according to the standard, P67 _should_ be used to enable USB external power, and to detect over-current situations.
However, we _wrongly_ use it to reset the external USB Hub. And to make the board work, this pin had to be taken from this regulator, and the regulator was set to always on to prevent other things from failing.
A regulator without output to actually control its output must be marked always-on, or does not make sense.